functions/_fifc_action.fish

```diff
@@ -53,7 +53,7 @@ function _fifc_action
             eval $$comp[$i][4]
             break
         else if test "$action" = source; and test -n "$$comp[$i][5]"
-            set _fifc_custom_fzf_opts "$$comp[$i][6]"
+            set fifc_custom_fzf_opts "$$comp[$i][6]"
```

Comment:
```suggestion

```

I fixed the issue above by removing this line. Can you clarify what this was meant to do?